mlflow/lightgbm.py

```diff
@@ -392,12 +399,6 @@ def log_feature_importance_plot(features, importance, importance_type):
                 plt.close(fig)
                 shutil.rmtree(tmpdir)
 
-        if not mlflow.active_run():
-            try_mlflow_log(mlflow.start_run)
-            auto_end_run = True
-        else:
-            auto_end_run = False
-
         original = gorilla.get_original_attribute(lightgbm, "train")
```

Comment:
We should call `original` directly rather than fetching it here. To catch this from a test perspective, can we add `mlflow.lightgbm` to https://github.com/mlflow/mlflow/blob/master/tests/autologging/test_autologging_safety_integration.py#L23?

Reply:
Done!